data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+		require "Plasma Cannon"
 		conversation
-			`As you are walking through the spaceport, a woman comes up to you and introduces herself as an engineer at one of the Remnant development labs. She pauses awkwardly before beginning to chant: "As your ship is the only one we have seen in many decades that has outfits from ancestral space, we are intrigued by it. Could we take a closer look?"`
+			`You remember that Taely was interested in seeing examples of technology from human space. Would you like to show her the plasma cannons you retrieved?`
 			choice
-				`	"Yes, you are welcome to examine it."`
-					goto deep
-				`	"Yes, but don't touch anything."`
-				`	"No, stay away from my ship."`
-					decline
+				`	Not now.`
+					defer
+				`	Yes.`
+			branch stuff
+				has "Remnant Tech Retrieval: done"
+			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely just finishing the installation of a thrasher cannon in a starling. "Greetings, <first>. I will be down in a minute." She does a few more things inside a panel, reseals the hatch, and slides down a fin to land next to you. "So, do you have something for me?"`
+				goto the_rest
+			label stuff
+			`	Remembering that she spends most of her time in the shipyard, you head straight there and have almost arrived when you are approached by another remnant who introduces herself as one of Taely's aides. "Greetings, Captain" she trills. "Our scans picked up indications of uncatalogued human weaponry. Were you bringing it to show us?"`
+			label the_rest
+			`	You lead her back to your ship, where you show her the plasma cannon. "I have never seen weapons designed to handle such a high heat output! Is this some kind of plasma based weapon?" she asks, gesturing at a cannon. You nod, and she continues, "Intriguing. We have noticed that the Korath ships seem to run particularly hot, and have speculated that overheating them might be an effective means of disabling them. I would enjoy researching these a bit more. If you could deliver two to <planet> for us you would be well compensated."`
+				accept
+	on complete
+		outfit "Plasma Cannon" -2
+		payment 1130000
+		"remnant met taely" ++
+		conversation
+			`As the <ship> settles onto the pad closest to the research facility, you can see a weapons specialist checking over a large cannon on a platform beneath a nearby Albatross. When he sees you starting to unload the two plasma cannons, he snaps the casing closed and gestures to someone out of sight. Within moments both him and an empty flatbed are there to pick up the weapons.`
+			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
+			branch wave
```

Comment:
Laughing at this branch, because if the "gesture" is really just a wave, I think I'd understand that without having to have learned some Remnant.